packages/terra-filter-view/src/terra-dev-site/doc/example/OptGroup.jsx

```diff
@@ -0,0 +1,16 @@
+import React from 'react';
+
+// eslint-disable-next-line import/no-extraneous-dependencies, import/no-unresolved, import/extensions
+import FilterView from 'terra-filter-view/lib/FilterView';
+
+const OptGroup = () => (
+  <FilterView variant="list" >
+    <FilterView.OptGroup label="Colors">
+      <FilterView.Option value="red" display="Red" key="red" />
+      <FilterView.Option value="blue" display="Blue" key="blue" />
+      <FilterView.Option value="green" display="Green" key="green" />
+    </FilterView.OptGroup>
+  </FilterView>
+);
+
+export default <OptGroup />;
```

Comment:
Is there a reason we're exporting this as a element instead of function?
(Why not use `export default OptGroup;`?)

Reply:
The [DocTemplate expects a React Element](https://github.com/cerner/terra-core/blob/master/packages/terra-doc-template/src/ExampleTemplate.jsx#L17) instead of a function

Reply:
We typically create the react element in the doc template page and export the function.

Reply:
Ah, I see, I can change that.

Reply:
There are other instances of odd exports in your doc/test pages, I didn't review them all incase you had a reason you needed to export like wrote it.

Reply:
will take a look and fix those